platforms/android/app/src/main/java/com/maq/xprize/cci/hindi/SplashScreenActivity.java

```diff
@@ -24,13 +24,23 @@
 import java.io.IOException;
 import java.util.zip.ZipFile;
 
+import static com.cci.DownloadExpansionFile.xAPKs;
 
-public class SplashScreenActivity extends Activity {
 
+public class SplashScreenActivity extends Activity {
+    public static SharedPreferences sharedPref;
     Intent intent = null;
+    String obbFilePath;
+    File obbFile;
+    ZipFile obbZipFile;
+    Zip zipFileHandler;
+    String dataFilePath;
+    File packageDir;
+    int mainFileVersion;
+    int patchFileVersion;
 
-    public static String getUnzippedExpansionFilePath() {
-        return "/storage/emulated/0/Android/data/com.maq.xprize.CCI.hindi/files/";
+    public static String getDataFilePath() {
+        return "/storage/emulated/0/Android/data/com.maq.xprize.cci.hindi/files/";

```

Comment:
Use getPackageName() to retrieve the package name.

Reply:
Updated